src/python/Utils/EmailAlert.py

```diff
@@ -16,9 +16,13 @@ class EmailAlert:
 
     def __init__(self, config):
```

Comment:
how about transforming this input parameter from a Configuration object to a flat dict? Then in the lines below, we could have something like:
```
        self.serverName = configDict.get("smtpServer", "localhost")
```
and so on.